src/NetworkStructures.jl

```diff
@@ -1,19 +1,21 @@
-"""
-    This module contains the logic that calculate the index structures
-    and data access structs that Network Dynamics makes use of.
+#
+# This file contains the logic that calculate the index structures
+# and data access structs that Network Dynamics makes use of.
+#
+# The key structure is the GraphData structure that allows accessing data on
+# vertices and edges of the graph in an efficient manner. The neccessary indices
+# are precomputed in GraphStructure.
 
-    The key structure is the GraphData structure that allows accessing data on
-    vertices and edges of the graph in an efficient manner. The neccessary indices
-    are precomputed in GraphStructure.
-"""
-module NetworkStructures
 
 using LightGraphs
 using LinearAlgebra
-export GraphStruct, GraphData, EdgeData, VertexData
 
 const Idx = UnitRange{Int}
 
+export GraphStruct, GraphData, EdgeData, VertexData
```

Comment:
We don't need to export for internal usage anymore because there is no more namespace separation. I think we can get rid of the exports
- GraphStruct
- GraphData
- EdgeData
- VertexData
- swap_v_array!
- swap_e_array!